packages/patternfly-4/react-core/src/components/Button/Button.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { Button, ButtonVariant } from './Button';
 import React from 'react';
-import { shallow } from 'enzyme';
+import { shallow, mount } from 'enzyme';
 import { CartArrowDownIcon } from '@patternfly/react-icons';
 
 Object.values(ButtonVariant).forEach(variant => {
```

Comment:
Change `shallow` to `mount` for snapshot tests.

Reply:
Done.